sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -84,31 +84,8 @@ The Language service supports both [multi-service and single-service access][ser
 resource if you plan to access multiple cognitive services under a single endpoint/key. For Language service access only,
 create a Language service resource.
 
-You can create either resource using the 
-
-**Option 1:** [Azure Portal][create_new_resource] 
-
-**Option 2:** [Azure CLI][azure_cli]
-
-Below is an example of how you can create a Language service resource using the CLI:
-
-```bash
-# Create a new resource group to hold the Language service resource -
-# if using an existing resource group, skip this step
-az group create --name <your-resource-group> --location <location>
-```
-
-```bash
-# Create language service
-az cognitiveservices account create \
-    --name <your-resource-name> \
-    --resource-group <your-resource-group-name> \
-    --kind TextAnalytics \
-    --sku <sku> \
-    --location <location> \
-    --yes
-```
-For more information about creating the resource or how to get the location and sku information see [here][azure_cli]
+You can create the resource using the [Azure Portal][create_new_resource_in_azure_portal] or 
+[Azure CLI][create_new_resource_in_azure_cli] following the steps in [this document][azure_cli_doc].
```

Comment:
are the links switched?

Reply:
Can you point me to which two are switched?

Reply:
```suggestion
You can create the resource using the [Azure Portal][create_new_resource_in_azure_portal] or 
[Azure CLI][azure_cli_doc] following the steps in [this document][create_new_resource_in_azure_cli].
```